python/ray/train/batch_predictor.py

```diff
@@ -231,6 +253,9 @@ def predict_pipelined(
             max_scoring_workers: If set, specify the maximum number of scoring actors.
             num_cpus_per_worker: Number of CPUs to allocate per scoring worker.
             num_gpus_per_worker: Number of GPUs to allocate per scoring worker.
+            separate_gpu_stage: If using GPUs, specifies whether to execute GPU
```

Comment:
Is there a case when we would not want this enabled?

Reply:
If the preprocessor is very lightweight, then enabling this could hurt more than it helps, due to excess disk spilling (in non-pipelined mode).